client/verta/verta/client.py

```diff
@@ -671,7 +671,12 @@ def get_or_create_registered_model(self, name=None, desc=None, labels=None, work
         id : str, optional
             ID of the registered_model. This parameter cannot be provided alongside `name`, and other
             parameters will be ignored.
-
+        actionType : str, optional
+            Action type of the registered_model.
+            OTHER, CLASSIFICATION, CLUSTERING, DETECTION, REGRESSION, TRANSCRIPTION, TRANSLATION
+        dataType : str, optional
+            Data type of the registered_model.
+            OTHER, AUDIO, IMAGE, TABULAR, TEXT, VIDEO
```

Comment:
Instead of `str`, could you create new `class`es for these?

They would be **very** similar to what we have for registry lock levels ([`client/verta/verta/registry/lock/`](https://github.com/VertaAI/modeldb/tree/b90a637/client/verta/verta/registry/lock)). So probably create new folders `registry/action_type/` and `registry/data_type/`, and copy the patterns used in lock level.

Reply:
it makes sense .. I created classes